.github/workflows/snap.yaml

```diff
@@ -0,0 +1,14 @@
+name: Snap
+
+on:
+  push: {}
+  pull_request: {}
+  workflow_dispatch: {}
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+    steps:
+    - uses: actions/checkout@v4
+    - uses: snapcore/action-build@v1
+      id: snapcraft
```

Comment:
Let's remove the ide since it isn't being referenced.

Reply:
Done